Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN poetry install --no-dev
 COPY ./todo_app ./todo_app
 ENTRYPOINT poetry run gunicorn --bind 0.0.0.0 'todo_app.app:create_app()'
 
-FROM base as tests
+FROM base as e2e_test_base
```

Comment:
Minor, but I've noticed my Docker has recently been telling me off more for having `as` in lowercase:
> - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)